ui/src/services/job/JobStatus.js

```diff
@@ -3,17 +3,17 @@ import { Enum, EnumValue } from "../enum/Enum";
 export const JobStatus = Enum({
   PENDING: EnumValue("pending", {
     label: "Pending",
-    color: "#F5A700",
+    color: "#FEC514",
```

Comment:
Updated the values of the status colours here with respect to those found here: https://eui.elastic.co/#/theming/colors/values